src/receipt/_style.scss

```diff
@@ -0,0 +1,67 @@
+#receipt-container {
+    width: 1170px;
+    padding-top: 2rem;
+}
+
+.page__receipt {
+    h2 {
```

Comment:
[curious/confirmation] Are the styles here relevant for _all_ h2 on the page?

Reply:
They should be, but weirdly I am modifying the size of 1 out of the 2 h2's with a specific px size. 

I think this is another case of "following the original" which is not a good explanation but I will change them to be different heading levels since they are different.